src/stacker/ConcreteStacker.ts

```diff
@@ -62,30 +63,62 @@ export class ConcreteStacker implements Stacker {
     const commitBranchPairs = await this.sourceControl.attachSttackBranchesToCommits(
       [commit],
     );
-    //TODO: Implemenet createOrUpdatePRForCommits with new parameters
-    // const commits = this.collaborationPlatform.createOrUpdatePRForCommits(commitBranchPairs);
+    const commitsWithMetaData: Array<{
+      commit: Commit;
+      headBranch: BranchName;
+      baseBranch: BranchName;
+    }> = [];
+    commitBranchPairs.forEach((commitBranchPair) => {
+      commitsWithMetaData.push({
+        commit: commitBranchPair.commit,
+        headBranch: commitBranchPair.sttackBranch,
+        baseBranch: "master",
+      });
+    });
```

Comment:
I think we should use `map` here (and below) instead? (you may need to run Prettier on this)
```suggestion
    const commitsWithMetaData = commitBranchPairs.map((commitBranchPair) => ({
      commit: commitBranchPair.commit,
      headBranch: commitBranchPair.sttackBranch,
      baseBranch: "master", // TODO: Implement retrieval of base branch for a given commit
    }));
```